### DISCUSSION TITLE
获取NTP时间失败如何解决？

### DISCUSSION THREAD

**Discussion by jd3jj83h91fh** [mikumifa/biliTickerBuy#445] [提问专区]
![无标题](https://github.com/user-attachments/assets/a9a20dbf-12dc-490d-9fdd-aa0a6dd80723)

**Comment by mikumifa** [comment]
阿里云要关梯子

**Reply by jd3jj83h91fh** [reply]
这个还要用阿里云？关闭梯子再试一次，还是这样。

**Reply by mikumifa** [reply]
> 这个还要用阿里云？关闭梯子再试一次，还是这样。

重启，这个就是连不上国内网原因

**Reply by jd3jj83h91fh** [reply]
重启之后，未开启梯子，直接打开程序，还是这样。这个是不是要用到阿里云的NTP服务器设置？我没弄。需要弄吗？
